src/components/FlawAffects/FlawAffectsTableRow.vue

```diff
@@ -188,13 +188,13 @@ function affectednessChange(event: Event, affect: ZodAffectType) {
         v-model="affect.affectedness"
         class="form-select"
         @keydown="handleKeystroke($event, affect)"
+        @change="affectednessChange($event, affect)"
```

Comment:
Think this is probably fine, but wanted to mention using `watch` is another option in case props/model would change for some reason.